package.json

```diff
@@ -18,27 +18,28 @@
     "i": "~0.3.2",
     "lodash": "3.10.1",
     "mkdirp": "0.5.1",
+    "mocha": "^3.0.0",
     "moment": "2.10.6",
-    "mongoose": "~3.8.21",
+    "mongoose": "^4.5.8",
```

Comment:
Lets better freeze the versions on exact values (as in `"4.5.8"`). At least for everything we explicitly upgrade to and test